src/twisted/internet/test/test_endpoints.py

```diff
@@ -1324,7 +1324,7 @@ def createServerEndpoint(self, reactor, factory, **listenArgs):
         address = IPv4Address("TCP", "0.0.0.0", 0)
 
         if listenArgs is None:
-            listenArgs = {}
+            listenArgs = {}  # type: ignore[unreachable]
```

Comment:
In order for `listenArgs` to be `None` here, it would have to be passed in as a keyword argument itself:

```python
>>> def createServerEndpoint(self, reactor, factory, **listenArgs):
...     pass
... 
>>> createServerEndpoint(None, None, None, listenArgs=None)
>>> 
```
 
It's a weird thing to do, and I don't see any uses of that in Twisted itself, but uh… ok.

Reply:
I checked in Python 3.6 and 3.8 (and even 2.7, out of curiosity), and when called with `listenArgs=None`, the value of `listenArgs` will be `{'listenArgs': None}` instead of `None`. So I think mypy is correct and the code is actually unreachable.

Reply:
Oh, I didn't think to look at what actually got handed in!  OK, I'll remove these lines.